lib/conversions/test/spec/SpecHook.hs

```diff
@@ -0,0 +1,6 @@
+module SpecHook where
+
+import Test.Hspec
+
+hook :: Spec -> Spec
+hook = parallel
```

Comment:
I wonder what's the point of this module? What is the problem that it solves?

Reply:
> I wonder what's the point of this module? What is the problem that it solves?

It makes all tests run in parallel by default. See https://hspec.github.io/parallel-spec-execution.html

Since there are a few packages that use a similar technique, I'll make a subsequent PR that adds documentation to all of those packages in one go.

Reply:
@Unisay
As promised, I've created a PR to document the reason that these modules exist: https://github.com/cardano-foundation/cardano-wallet/pull/4156